tests/DocumentationExamplesTest.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests;
+
+use App\Http\Controllers\RelationshipController;
+use App\Models\Cargo;
+use App\Models\Moon;
+use App\Models\Orbit;
+use App\Models\Passenger;
+use App\Models\Planet;
+use App\Models\SpaceExplorer;
+use App\Models\SpaceShip;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\SQLiteBuilder;
+use Illuminate\Support\Facades\Schema;
+
+use function assert;
+
+class DocumentationExamplesTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testOneToOne(): void
+    {
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/one-to-one/Orbit.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/one-to-one/Planet.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/RelationshipController.php';
+
+        // Clear the database
+        Planet::truncate();
+
+        $controller = new RelationshipController();
+        $controller->oneToOne();
+
+        $planet = Planet::first();
+        $this->assertInstanceOf(Planet::class, $planet);
+        $this->assertInstanceOf(Orbit::class, $planet->orbit);
+    }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testOneToMany(): void
+    {
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/one-to-many/Planet.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/one-to-many/Moon.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/RelationshipController.php';
+
+        // Clear the database
+        Planet::truncate();
+
+        $controller = new RelationshipController();
+        $controller->oneToMany();
+
+        $planet = Planet::first();
+        $this->assertInstanceOf(Planet::class, $planet);
+        $this->assertCount(2, $planet->moons);
+        $this->assertInstanceOf(Moon::class, $planet->moons->first());
+
+        $controller->planetMoonsDynamic();
```

Comment:
This and a number of other tests don't run any assertions after the controller call. I assume we only care about the controller not throwing an error. If so, we should note this in a comment here to avoid confusion.

We can also consider using `assert` in the controller outside of the doc comments to at least verify that data was correctly loaded, assuming code outside of the comments doesn't get shown on the docs page.

Reply:
I merged the doc examples with the test cases, similar to [library's `DocumentationExamplesTest`](https://github.com/mongodb/mongo-php-library/blob/master/tests/DocumentationExamplesTest.php) but in the `./docs/includes` dir. Adding a new Test Suite in the PHPUnit config.